Library/Adjustment.cs

```diff
@@ -236,7 +244,7 @@ internal override void WriteXml(XmlTextWriter xmlWriter)
             xmlWriter.WriteStartElement("adjustment"); 
             xmlWriter.WriteElementString("description", Description);
             xmlWriter.WriteElementString("unit_amount_in_cents", UnitAmountInCents.AsString());
-            xmlWriter.WriteElementString("currency", Currency);
+            //if (!Currency.IsNullOrEmpty()) xmlWriter.WriteElementString("currency", Currency);
```

Comment:
Should probably add a new serializer that doesn't render currency the same way we did for `Subscription`.